AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTests.m

```diff
@@ -227,6 +228,41 @@ - (void)testApplyEnabledStateWorks {
   [[MSAnalytics sharedInstance].sessionTracker stop];
 }
 
+- (void)testApplyEnabledStateWithAutoPageTrackingEnabled {
```

Comment:
What do you want to test here?

Reply:
I'm testing the case where autoPageTracking is enabled and missedPageViewName returns not nil, then trackPage should be called with missedPageViewName.

Reply:
This makes sense to me.

Reply:
The reason for the comment is that the method name is not quite descriptable for its testing. Can you rename?